frontend/src/components/CourseSearchBar/CourseSearchBar.tsx

```diff
@@ -15,6 +15,7 @@ type SearchResultLabelProps = {
   runMutate?: (courseId: string) => void;
 };
 
+/* eslint-disable */
```

Comment:
😭

Reply:
If were up to me, this would be in root :)

(i'll remove)

Reply:
>:(